inline.h

```diff
@@ -3090,17 +3090,21 @@ Perl_isUTF8_CHAR_flags(const U8 * const s0, const U8 * const e, const U32 flags)
 
 /*
 
-=for apidoc is_utf8_valid_partial_char
+=for apidoc      is_utf8_valid_partial_char
+=for apidoc_item is_utf8_valid_partial_char_flags
 
-Returns 0 if the sequence of bytes starting at C<s> and looking no further than
-S<C<e - 1>> is the UTF-8 encoding, as extended by Perl, for one or more code
-points.  Otherwise, it returns 1 if there exists at least one non-empty
-sequence of bytes that when appended to sequence C<s>, starting at position
-C<e> causes the entire sequence to be the well-formed UTF-8 of some code point;
-otherwise returns 0.
+These each return 0 if the sequence of bytes starting at C<s> and looking no
+further than S<C<e - 1>> is the UTF-8 encoding for one or more code points.
+That is, 0 is returned if C<s> points to at least one entire UTF-8 encoded
+character.
 
-In other words this returns TRUE if C<s> points to a partial UTF-8-encoded code
-point.
+Otherwise, they return 1 if there exists at least one non-empty sequence of
+bytes that when appended to sequence C<s>, starting at position C<e> causes the
+entire sequence to be the well-formed UTF-8 of some code point; otherwise
+returns 0.
```

Comment:
Sentence above contains two instances of `otherwise`.  The part after the semicolon can probably be cut.

Reply:
done